lib/scrabble/scrabble.rb

```diff
@@ -0,0 +1,154 @@
+
+module Scrabble
+
+	LETTER_VALUES = [
+		[1, "a", "e", "i", "o", "u", "l", "n", "r", "s", "t"],
+		[2, "d", "g"],
+		[3, "b", "c", "m", "p"],
+		[4, "f", "h", "v", "w", "y"],
+		[5, "k"],
+		[8, "j", "x"],
+		[10, "q", "z"]
+	]
+	
+
+	class Scrabble
+
+		def self.value_of(letter)
+			LETTER_VALUES.each do |value_letters_array|
+				if value_letters_array.include?(letter)
+					return value_letters_array[0]
+				end
+			end
+		end
+
+		def self.score(word)
+			word = word.to_s.downcase
+
+			word_score = 0
+
+			word_array = word.chars.to_a
+
+			if word_array.length > 7
+				return "Please enter a word that has 7 or fewer characters."
+
+			elsif word_array.include?(" ")
+				return "Please only enter one word."
+
+			elsif word_array.length == 0
```

Comment:
ruby strings actually support `length` and `include?` and calling these methods on `word` would return the same results as calling these methods on `word_array`. by doing that you could skip line 30 entirely and just call `word.chars.each` on line 42, removing an extra variable and simplifying things a little bit.